src/apps/frontend/pages/authentication/otp/index.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import toast from 'react-hot-toast';
+import { useNavigate } from 'react-router-dom';
+
+import { Button, H2, VerticalStackLayout } from '../../../components';
+import routes from '../../../constants/routes';
+import { AsyncError } from '../../../types';
+import { ButtonKind } from '../../../types/button';
+import useTimer from '../../../utils/use-timer.hook';
+import AuthenticationFormLayout from '../authentication-form-layout';
+import AuthenticationPageLayout from '../authentication-page-layout';
+
+import OTPForm from './otp-form';
+
+export const OTPPage: React.FC = () => {
+  const sendOTPDelayInMilliseconds = 60_000;
```

Comment:
Let's store it in global constants file.

Reply:
stored in global constant and used it here